docs/guides/server-side-rendering.md

```diff
@@ -10,6 +10,25 @@ This document will demonstrate how to achieve Server Side Rendering (SSR) with [
 
 > This document assumes that you already have a Next.js application in place.
 
+```javascript
+---
+embed: true
+---
+<ToggleBlockquote
+  summary="Note!"
+>
+  <ToggleBlockquote.Paragraph>
+   You should not install the InstUI meta package <code>@instructure/ui</code> since it has components that are not yet Server Side Renderable out of the box.
```

Comment:
I would mention that we are planning to solve this in V9, but it would be a breaking change now

Reply:
Fixed